torch/csrc/distributed/c10d/TCPStoreLibUvBackend.cpp

```diff
@@ -31,7 +31,10 @@ Other callbacks don't provide exception safety so avoid there.
 
 */
 
-#define DEFAULT_BACKLOG 16384
+// This controls how many un-accepted TCP connections can be waiting in the
+// backlog. This should be at least world size to avoid issues on init. We set
+// it to -1 to use the host max value which is controlled by `soconnmax`.
+#define DEFAULT_BACKLOG -1
```

Comment:
@XilunWu do you remember why we picked 16k? iirc this was changed by @kumpera early on in the libuv backend development, maybe we just didn't imagine needing larger than this.

Reply:
This DEFAULT_BACKLOG setting seems to be following the libuv example -- likely they didn't realize that -1 worked to default to max and just set it to be the max known world size? 

https://github.com/libuv/libuv/blob/v1.x/docs/code/tcp-echo-server/main.c#L7